routes/index.js

```diff
@@ -36,7 +36,10 @@ module.exports = function () {
   function redirect(path, base='https://old.datahub.io') {
     return function(req, res) {
       let dest = base + path;
-      if (req.params[0]) {
+      if (req.params[0] && Object.keys(req.query).length > 0) {
+        let query_string = '?' + req.url.split('?')[1]
```

Comment:
Can you rename `query_string` to `queryString` as in JS we use camelcase when naming variables?